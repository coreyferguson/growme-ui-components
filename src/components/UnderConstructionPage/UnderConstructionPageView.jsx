import React from 'react';

export default function UnderConstructionPageView() {
  return (
    <React.Fragment>
      <h1>under construction</h1>
      <div className='under_construction'></div>
    </React.Fragment>
  );
};