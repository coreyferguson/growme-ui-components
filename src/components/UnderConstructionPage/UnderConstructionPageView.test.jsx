import React from 'react';
import UnderConstructionPageView from './UnderConstructionPageView.jsx';
import renderer from 'react-test-renderer';

describe('UnderConstructionPageView', () => {
  test('snapshot', () => {
    const tree = renderer
      .create(<UnderConstructionPageView />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
