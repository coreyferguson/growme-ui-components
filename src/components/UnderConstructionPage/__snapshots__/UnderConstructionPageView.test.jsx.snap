// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnderConstructionPageView snapshot 1`] = `
Array [
  <h1>
    under construction
  </h1>,
  <div
    className="under_construction"
  />,
]
`;
